app/services/publication-service.js

```diff
@@ -56,23 +56,23 @@ export default class PublicationService extends Service {
   ) {
     const now = new Date();
 
-    let case_;
+    let _case;
```

Comment:
Not a comment on this line, but on `getIsViaCouncilOfMinisters()` in the `publication-service`.

The logic to check whether a publication flow is via council of ministers becomes simpler. We just need to check whether the case related to the publication-flow has a related decisionmaking-flow. Might be that this helper function `getIsViaCouncilOfMinisters()` becomes void.